azext_iot/tests/test_iot_central_int.py

```diff
@@ -475,6 +521,40 @@ def _delete_user(self, user_id) -> None:
             checks=[self.check("result", "success")],
         )
 
+    def _create_cde(self,):
+
+        cde = []
+        for source in DataSourceType:
+            display_name = self.create_random_name(prefix="aztest", length=24)
+            export_id = display_name
+            command = "iot central app cde create --app-id {} --dn {} -d {} -s {} -e False -t {} -c {}  --en {} ".format(
```

Comment:
Why not create the CDE as enabled since there is a real storage account?

Reply:
Status of the  CDE could be in one of the two possible states  - starting / running. Validation will have to account for possible states and will introduce slight complexity.  From a testing standpoint running state of stopped is more deterministic way of validation.